tests/unit/sqlalchemy/test_compiler.py

```diff
@@ -47,7 +35,10 @@ def dialect():
 def test_limit_offset(dialect):
     statement = select(table).limit(10).offset(0)
     query = statement.compile(dialect=dialect)
-    assert str(query) == 'SELECT "table".id, "table".name \nFROM "table"\nOFFSET :param_1\nLIMIT :param_2'
+    assert (
+        str(query)
+        == 'SELECT "table".id, "table".name \nFROM "table"\nOFFSET :param_1\nLIMIT :param_2'
+    )
```

Comment:
questionable redability

Reply:
A multi line string would be more fitting here. However that triggers flake8 as it seems flake8 doesn't allow spaces at EOL, even within multi line strings.